src/adi_evaluation.R

```diff
@@ -0,0 +1,71 @@
+# Load packages
+library(dplyr)
+
+# Set working directory to avoid wrong PATHs
```

Comment:
I think it is a local path on your environment. We should not push this to the remote repo